types/graphql-depth-limit/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This should just be `{ "extends": "dtslint/dt.json" }`.

Reply:
Works for me -- Try reinstalling `node_modules` (removing `package-lock.json` too)? Or just changing `tslint.json` and pushing to see what travis shows?

Reply:
Tnx worked now, once I re-installed node modules. And pushed and travis is green also now :)